sql/moz-fx-data-shared-prod/fenix_derived/profile_dau_metrics_marketing_geo_testing_v1/query.sql

```diff
@@ -0,0 +1,60 @@
+SELECT
+  active_users.submission_date,
+  active_users.normalized_channel,
+  active_users.app_name,
+  active_users.app_display_version AS app_version,
+  COALESCE(active_users.country, '??') AS country,
+  active_users.city,
+  active_users.geo_subdivision,
+  active_users.locale,
+  active_users.os,
+  active_users.os_version,
```

Comment:
`os` and `os_version` need to be `normalized_os` and `normalized_os_version` respectively

Reply:
you are correct. Updated this.